src/systems/eu/eu-component/eu-component-global-navigation/eu-component-global-navigation.scss

```diff
@@ -0,0 +1,177 @@
+/**
+* Global navigation component
+* @define global-navigation
+*/
+
+.ecl-global-navigation {
+  @include ecl-hidden-print();
+
+  background-color: map-get($ecl-colors, 'grey-10');
+  font-size: map-get($ecl-font-size, 's');
+  margin: 0;
+}
+
+.ecl-global-navigation__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ecl-global-navigation__option {
+  flex-grow: 1;
+}
+
+.ecl-global-navigation__link {
+  color: map-get($ecl-colors, 'grey-100');;
+  display: block;
+  line-height: 1.3;
+  padding: map-get($ecl-spacing, 'xxs') map-get($ecl-spacing, 's');
+  text-decoration: none;
+  width: 100%;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: map-get($ecl-colors, 'grey-100');;
+  }
+}
+
+.ecl-global-navigation__link--official {
+  align-items: center;
+  display: flex;
+  position: relative;
+
+  .ecl-global-navigation__label {
+    padding-left: map-get($ecl-spacing, 'xl');
+  }
+
+  &::before {
+    background: url($ecl-assets-path+'images/logo.svg') no-repeat;
+    background-size: contain;
+    content: '';
+    display: block;
+    height: 25px;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 37px;
+  }
+}
+
+.ecl-global-navigation__link--institution {
+  background-color: map-get($ecl-colors, 'grey-25');
+}
+
+.ecl-global-navigation__toggle {
+  background-color: map-get($ecl-colors, 'grey-25');
+  padding: map-get($ecl-spacing, 's') map-get($ecl-spacing, 'm');
```

Comment:
I think the `padding-bottom` should be higher on desktop (comparing the result with the mockup)